src/strategies/DeltaNeutralLp.sol

```diff
@@ -84,25 +100,44 @@ contract DeltaNeutralLp is AccessStrategy {
         borrowPrice = uint256(price);
     }
 
-    /// @notice Get price of WETH in USDC (borrowPrice) from Sushiswap. Has 8 decimals.
+    /**
+     * @notice Get price of WETH in USDC (borrowPrice) from Sushiswap. Has 6 decimals.
```

Comment:
Just use `@dev` when commenting on internal functions.

Reply:
`Get price of borrow denominated in asset from sushiswap`. Remove the part about 6 decimals.